### PR TITLE
Add scrolling to table, use table widget for support bundle

### LINKED ISSUE #220: Bug: scrolling collectors/analyzers in CLI with arrows does not work
On small windows where there are more preflight checks than fit on the screen, scrolling down selects checks and displays them on the right, but they're hidden on the left.

### PR DESCRIPTION
**Pull request by GraysonNull**

### CODE REVIEW

File: cmd/troubleshoot/cli/interactive_results.go [hunk 1]
```diff
@@ -142,11 +128,143 @@ func drawFooter() {
...
+		currentTop = currentTop + height + 1
+	}
+}
+
+func estimateNumberOfLines(text string, width int) int {
+	lines := len(text)/width + 1
+	return lines
+}
+
+// func drawGrid(analyzeResults []*analyzerunner.AnalyzeResult) {
```
Comment (marccampbell) [inline cmd/troubleshoot/cli/interactive_results.go]: What's this unused block for?

File: cmd/troubleshoot/cli/interactive_results.go [hunk 2]
```diff
@@ -142,11 +128,143 @@ func drawFooter() {
...
+// 		}
+// 	}
+// }
+
 func showSaved(filename string) {
 	termWidth, termHeight := ui.TerminalDimensions()
 
 	savedMessage := widgets.NewParagraph()
-	savedMessage.Text = fmt.Sprintf("Preflight results saved to\n\n%s", filename)
+	savedMessage.Text = fmt.Sprintf("Support Bundle analysis results saved to\n\n%s", filename)
```
Comment (marccampbell) [inline cmd/troubleshoot/cli/interactive_results.go:223]: nice catch

File: cmd/troubleshoot/cli/interactive_results.go [hunk 3]
```diff
@@ -198,3 +316,19 @@ func save(analyzeResults []*analyzerunner.AnalyzeResult) (string, error) {
 
 	return filename, nil
 }
+
+func appName(supportBundleName string) string {
```
Comment (marccampbell) [inline cmd/troubleshoot/cli/interactive_results.go]: This function looks to be copied from elsewhere? Should we move it to a shared package and use the same implementation?

**Review by marccampbell — COMMENTED** [review]
@GraysonNull Can you attach a couple of screenshots showing the previous and the new terminal UI here for review?

File: cmd/troubleshoot/cli/interactive_results.go (hunk 1 above)
Comment (GraysonNull) [reply cmd/troubleshoot/cli/interactive_results.go]: oh shoot, nothing. Just need to remove it

File: cmd/troubleshoot/cli/interactive_results.go (hunk 3 above)
Comment (GraysonNull) [reply cmd/troubleshoot/cli/interactive_results.go]: It is yes. Yep I can move it.

**Comment by GraysonNull** [comment]
https://www.loom.com/share/7879390bb56e4ee3a7eb838bf3c54430

**Review by marccampbell — APPROVED** [review]

**Pull request merged**